### PR TITLE
hook up choropleth client

### PR DESCRIPTION
**Pull request by chejennifer**
If chart_config.json has configs with isChoropleth set to true, choropleth charts will be shown in place explorer on the topic page
Changes:
- In place.ts, add extra promises to get geojson data and statvar/denom data
- added props in MainPane, ChartBlock, and Chart to pass the extra data
- In Chart, updated processData and drawChart with cases for choropleth chart

### CODE REVIEW

File: static/js/place/chart.tsx [hunk 1]
```diff
@@ -70,6 +77,14 @@ interface ChartPropType {
    * Scale number
    */
   scaling?: number;
+  /**
+   * Geojson data for places one level down of current dcid.
+   */
+  geoJsonData?: unknown;
```
Comment (shifucun) [inline static/js/place/chart.tsx:83]: would be nice to use the actual type for geoJson

File: static/js/place/chart.tsx [hunk 2]
```diff
@@ -114,12 +130,26 @@ class Chart extends React.Component<ChartPropType, ChartStateType> {
...
       ? "(" + this.props.snapshot.date + ")"
       : "";
-    const sources = this.props.trend
-      ? this.props.trend.sources
-      : this.props.snapshot.sources;
-    const exploreUrl = this.props.trend
-      ? this.props.trend.exploreUrl
-      : this.props.snapshot.exploreUrl;
+    let sources = [];
+    if (this.props.chartType !== chartTypeEnum.CHOROPLETH) {
```
Comment (shifucun) [inline static/js/place/chart.tsx:134]: does choropleth data have sources now?

**Review by shifucun — COMMENTED** [review]
looks mostly good, just a few nits

File: static/js/place/chart.tsx (hunk 2 above)
Comment (chejennifer) [reply static/js/place/chart.tsx:134]: not yet, that's something I still need to add. Also need to get the exploreURL when getting choropleth data

**Review by beets — CHANGES_REQUESTED** [review]
nice change, just add a few todos

File: static/js/place/chart.tsx (hunk 1 above)
Comment (beets) [reply static/js/place/chart.tsx:83]: we can also add a type for it first (which is set to unknown for now) with a todo to add the complete type later

File: server/routes/api/chart.py [hunk 3]
```diff
@@ -371,6 +371,8 @@ def geojson(dcid):
                 choropleth_api.coerce_geojson_to_righthand_rule(
                     geojson['coordinates'], geojson['type']))
             features.append(geo_feature)
+    print("here\n")
```
Comment (shifucun) [inline server/routes/api/chart.py:374]: remove logging?

**Review by shifucun — APPROVED** [review]

**Review by shifucun — APPROVED** [review]

**Review by beets — APPROVED** [review]
thanks! let's get this in

**Pull request merged**